ss58-registry.json

```diff
@@ -370,6 +370,14 @@
 			"standardAccount": "*25519",
 			"website": "https://chainx.org/"
 		},
+		{
+			"prefix": 45,
+			"displayName": "UniArts Network",
+			"symbols": ["UART", "UINK"],
+			"decimals": [12, 12],
+			"standardAccount": "*25519",
```

Comment:
This is missing the "network" key, see https://github.com/uni-arts-chain/substrate/blob/9723141d4e5b6c6b5006809fdfaf76a189b9588f/ss58-registry.json#L5

Reply:
Damnit, that's my fault for missing in review. I assume it's `uniarts`, @tuminfei - can you confirm and make a followup PR?

Reply:
Oh @jacogr @joepetrowski , sorry I missed that, I will add it.